src/main/resources/assets/tconstruct/book/materials_and_you/sections/modifiers.json

```diff
@@ -1,10 +1,90 @@
 [
+  {
+    "type": "modifier",
+    "data": "modifiers/antiaquatic.json"
```

Comment:
Alphabetical is again a bit awkward here. Maybe we can sort them into groups? Ask alexbegt if we can bold something in the index, or otherwise make it look
**Like a**
* List title

**As in bold**
* And not bold bullet points

I suggest writing those header pages either way, lets us better explain bonus upgrade slots and ability slots on a page.

----

One potential list of groups:
* General (diamond, emerald, soulbound, etc.)
* Harvest (haste, magnetic)
* Weapon (fiery, necrotic, sharpness)
* Bonus upgrades (recapitated, harmonious, etc.)
* Slotless modifiers? (overslime, worldbound)
* Ability (luck, silk touch, draconic I suppose)